docs/modules/ROOT/pages/manage/relayers.adoc

```diff
@@ -452,14 +452,80 @@ Defender carefully tracks the costs of transactions sent to a relayer. In instan
 
 An "insufficient funds" error will throw when: `txCost > predictedBalance`.
 
-[[concurrency-and-rate-limiting]]
-== Concurrency and Rate Limiting
+[[transaction-throughput-and-load-balancing]]
+== Transaction Throughput and Load Balancing
 
 Relayers assign nonces atomically which allows them to handle many concurrent transactions. However, there do exist limits to optimize the infrastructure (all numbers below are cumulative of all Relayers in an account):
 
 * 100 total requests/second
 * 10 transactions/second
 
+For better reliability of the relayers, we recommend sending no more than *50 transactions/min* on a single relayer especially on fast moving chains like Polygon, Optimism, Arbitrum etc.. For example, if you want 250 transactions/min throughput, you would need to load balance across 5 relayers. These 5 relayers can be part of the same account.
+
+You may use the https://www.npmjs.com/package/@openzeppelin/defender-sdk[`Defender SDK`, window=_blank] package to load balance across multiple relayers. Here is a simple example on how you can do this:
+
+```ts
+require('dotenv').config();
+
+const { Defender } = require('@openzeppelin/defender-sdk');
+
+async function loadbalance() {
+  const LOAD_BALANCE_THRESHOLD = 50;
+  const relayerCredsForMainNet = [
+    {
+      relayerApiKey: process.env.RELAYER_API_KEY_1,
+      relayerApiSecret: process.env.RELAYER_API_SECRET_1,
+    },
+    {
+      relayerApiKey: process.env.RELAYER_API_KEY_2,
+      relayerApiSecret: process.env.RELAYER_API_SECRET_2,
+    },
+  ];
+  const relayerClientsForMainNet = relayerCredsForMainNet.map((creds) => new Defender(creds));
+
+  const getNextAvailableRelayer = async () => {
+    for (const client of relayerClientsForMainNet) {
+      const relayerStatus = await client.relaySigner.getRelayerStatus();
+      if (relayerStatus.numberOfPendingTransactions < LOAD_BALANCE_THRESHOLD) {
+        return client;
+      }
+      console.log(
+        `${relayerStatus.relayerId} is busy. Pending transactions: ${relayerStatus.numberOfPendingTransactions}/${LOAD_BALANCE_THRESHOLD}`,
+      );
```

Comment:
I think this is fine for now, better than nothing :) but yeah we should iterate on this to throw a warning if possible just like aws sdk v2 maintenance mode message but this goes back to adding validation of 50 txs/min on sdk side. We don't have to do this now we can wait for virtual relayers since that would solve that usecase.